src/booking/Offers/OfferTypes.ts

```diff
@@ -127,6 +127,13 @@ export interface Offer {
    * The ISO 8601 datetime at which the offer was last updated
    */
   updated_at: string
+
+  /**
+   * Whether this is a partial or full offer.
+   * A partial offer can't be booked directly, but it can be combined with other partial offers to form a full offer.
+   * Partial offers are only ever returned through the multi-step search flow.
+   */
+  partial: boolean
```

Comment:
Is this a required field now?

Reply:
rquired where? this field comes back in the response and is populated on full offers and partial offers.